package-lock.json

```diff
@@ -1,13100 +1,8 @@
 {
   "name": "contracts",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
Why are we reverting it to the previous version? @ashish10677

Reply:
I have not manually changed anything in `package-lock.json`. I just did an `npm install` and it got changed.